client/signup/steps/user-type/style.scss

```diff
@@ -0,0 +1,65 @@
+.user-type__wrapper {
```

Comment:
I see we're waiting for #31400 to apply the changes here too, but in any case, let's avoid duplicating styles. In any case where we're reusing the same thing, let's introduce a reusable component that we can all benefit from.